core/sup_tree_core/executor_pool/async_job_broker.ex

```diff
@@ -107,6 +107,23 @@ defmodule AntikytheraCore.ExecutorPool.AsyncJobBroker do
   end
 
   defp ensure_queue_added(%{queue_name: queue_name}) do
+    try do
+      groups = RaftFleet.consensus_groups() |> Map.keys()
+      if Enum.member?(groups, queue_name) do
```

Comment:
Why don't you use `Map.has_key?/1`?
Constructing a list of keys and traversing the list are both `O(n)` and unnecessary.

Reply:
I should have used `Map.has_key?/1`.
I fixed it in 3a3f71e5f1d76588b058f324ee419ad8d8aa9216